src/main.rs

```diff
@@ -2041,18 +2041,6 @@ mod tests {
                 0
             );
 
-            // Further test that we're MSI only now
-            aver_eq!(
-                tb,
-                guest
-                    .ssh_command("cat /proc/interrupts | grep -c 'IO-APIC'")
-                    .unwrap_or_default()
-                    .trim()
-                    .parse::<u32>()
-                    .unwrap_or(1),
-                0
-            );
```

Comment:
Maybe instead of removing it, we could make this test only being tested when the build does not include `acpi`.

Reply:
We don't build or run the tests without ACPI.

Reply:
Yes I know, but shouldn't we? Because we still want to make sure everything works when ACPI is compiled out, right?